cmd/atlascmd/schema.go

```diff
@@ -114,6 +116,20 @@ const (
 	answerAbort = "Abort"
 )
 
+// selectEnv is a pre-run hook for a cobra.Command that loads activeEnv with
+// the environment config from the current directory project file.
+func selectEnv(_ *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return nil
+	}
+	env, err := LoadEnv(projectFileName, args[0])
+	if err != nil {
+		return err
+	}
+	activeEnv = env
+	return nil
+}
```

Comment:
Why do we need a global var here? You can return the value instead.

Reply:
similar to cli flags, this value is used by other commands at runtime.

Reply:
Does not feel right to me. Can we change it, please?